src/tasks/tasks.py

```diff
@@ -455,6 +455,87 @@ def update_metrics(self, logits, labels, tagmask=None):
         self.scorer2(logits, labels)
         return
 
+@register_task('cola_probing', rel_path='CoLA_probing_npi/')
+class CoLAPTask(SingleClassificationTask):
```

Comment:
Use 'Probing', spelled out' in the class name, and add a comment explaining the difference from CoLATask.